pkg/cli/admin/inspect/namespace.go

```diff
@@ -74,12 +75,28 @@ func (o *InspectOptions) gatherNamespaceData(baseDir, namespace string) error {
 	}
 
 	klog.V(1).Infof("    Gathering pod data for namespace %q...\n", namespace)
+	nodeList, err := o.kubeClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
Just a nit: You could put the `nodeList` code under `if pods := resourcesToStore[corev1.SchemeGroupVersion.WithResource("pods")]; pods != nil {` condition as that's where it's used. Just in case the inspect command will not inspect pods resource to avoid listing the nodes unnecessarly.

Reply:
Good idea. Updated. Thank you for careful review :).